judge/judgedaemon.main.php

```diff
@@ -403,6 +404,7 @@ function fetch_executable(
             if ($retval!==0) {
                 return [null, "Failed to build executable in $execdir."];
             }
+            chmod($execrunpath, 0755);
```

Comment:
instead of doing this here let's do `chmod +x run` in the corresponding java and python scripts above

Reply:
Thank you for your review! Why though? I thought it made more sense to do it when the executable is created, the same way it is done with the build file once it is created (being consistent with that).

Reply:
Sounds reasonable